packages/babel-helpers/src/helpers.js

```diff
@@ -417,6 +562,30 @@ helpers.interopRequireWildcard = template(`
   })
 `);
 
+helpers.makeElementDescriptor = template(`
+  (function (kind, key, isStatic, descriptor, finisher) {
+    return { kind, key, isStatic, descriptor, finisher };
+  });
+`);
+
+//TODO
```

Comment:
Would be nice to include in the comment here that the details of merging are still up for discussion in TC39.

Reply:
The merging here seems like just overwriting and choosing the last one. I think merging should also include merging getter/setter pairs.

Reply:
@littledan, I have a question. Does merging also mean that if by a previous decorator you have some attribute, then if it isn't provided by a later decorator with an extra for the same key, then we just use the attribute from the previous extra object? For example, if enumerability is set to `false` for an extra with key "foo" by a previous decorator, then a later decorator needn't provide a enumerability value for an extra with key "foo", and we'll just use the previous enumerability value?

Current behaviour in this PR expects that all extras are well-formed descriptors, which means that we simply overwrite and choose the last one, as you noticed.